evalml/tests/component_tests/test_target_encoder.py

```diff
@@ -157,20 +157,6 @@ def test_get_feature_names():
     np.testing.assert_array_equal(encoder.get_feature_names(), np.array(['col_1', 'col_2', 'col_3']))
 
 
-def test_custom_indices():
```

Comment:
Moved this to `evalml/tests/pipeline_tests/regression_pipeline_tests/test_regression.py` since it was using a pipeline and is beyond just a component test. I'm honestly not too sure what this test is testing for... just that custom indices work don't break pipelines? 🤷